### PR TITLE
groovy update: regenerate

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: src/auto/settings.rs [hunk 1]
```diff
@@ -77,6 +77,10 @@ pub trait SettingsExt: 'static {
         gtk_application_prefer_dark_theme: bool,
     );
 
+    fn get_property_gtk_cursor_aspect_ratio(&self) -> f32;
```
Comment (sdroege) [inline src/auto/settings.rs:80]: No `feature = "v3_24_XX"`?

**Review by sdroege — COMMENTED** [review]
Looks good to me otherwise

File: src/auto/settings.rs (hunk 1 above)
Comment (sdroege) [reply src/auto/settings.rs:80]: This a) needs annotations in the C code, and b) configured in the `Gir.toml`

Comment (bilelmoussaoui) [reply src/auto/settings.rs:81]: It was added in Gtk 3.24.11 https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/NEWS#L90

**Comment by sdroege** [comment]
@bilelmoussaoui Please regenerate with the merged version of `gir-files` and then let's get it in :)

File: src/auto/settings.rs (hunk 1 above)
Comment (sdroege) [reply src/auto/settings.rs:81]: Can you (generally) also link to the upstream MR (or issues) you created for these things?

**Review by sdroege — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**